spec/system/admin/reports/sales_tax/sales_tax_totals_by_order_spec.rb

```diff
@@ -466,13 +466,22 @@ def extract_xlsx_rows(file, range)
       end
 
       context "pdf files" do
+        let!(:report_file_pdf) do
+          PDF::Reader.new("spec/fixtures/reports/sales_tax_by_order/sales_tax_by_order.pdf")
+        end
+
         it 'downloads the file' do
           expect(downloaded_filenames.length).to eq(0) # downloads folder should be empty
           select "PDF", from: "report_format"
           click_on "Go"
           wait_for_download
           expect(downloaded_filenames.length).to eq(1) # downloads folder should contain 1 file
           expect(downloaded_filename).to match(/.*\.pdf/)
+          downloaded_file_pdf = PDF::Reader.new(downloaded_filename)
+          # comparing pdf md5
+          md5_fixture = `cat report_file_pdf | grep -a -v "/CreationDate" | md5sum`.strip
+          md5_downloaded = `cat downloaded_file_pdf | grep -a -v "/CreationDate" | md5sum`.strip
+          expect(md5_fixture).to eql md5_downloaded
```

Comment:
Instead of comparing the md5sum, maybe we can compare the text content (or raw content) of the pages.

Example from the doc:
```ruby
    reader = PDF::Reader.new("somefile.pdf")

    reader.pages.each do |page|
      puts page.fonts
      puts page.text
      puts page.raw_content
    end
```